### DISCUSSION TITLE
Issues related to next-css in next 9.3.6 version

### DISCUSSION THREAD

**Discussion by ebs-PriyankaKanwar** [vercel/next.js#12388] [Help]
As i update next version for 9.3.6, Compiler starts warning to remove the " const withCSS = require('@zeit/next-css')"  globally and to import all css at app.js. After removing, i start getting issues in my production build. There are two css loading instead of one minified and this is creating some issues related to designs in anchor's, font Family and background colors. here is my next.config.js file :

const withFonts = require('next-fonts');
const withImages = require('next-images');
//  const withCSS = require('@zeit/next-css')

const withPlugins = require('next-compose-plugins');
const nextEnv = require('next-env');
const dotenvLoad = require('dotenv-load');
 
dotenvLoad();

withFonts({
  webpack(config, options) {
    // config.plugins.push(new webpack.EnvironmentPlugin(localEnv))
    return config;
  }
})

module.exports = withPlugins([
 
  nextEnv({
    staticPrefix: 'CUSTOM_STATIC_',
    publicPrefix: 'CUSTOM_PUBLIC_',
  }),withFonts, withImages
]);

- OS: [e.g. Ubantu]
- Browser[chrome]
- Version of Next.js:  "^9.1.6"

## Additional context

![Screenshot from 2020-05-01 18-29-31](https://user-images.githubusercontent.com/64646499/80821478-6f13b680-8bf6-11ea-8005-8e48d7e4f1df.png)

**Comment by ebs-PriyankaKanwar** [comment]
Hello team 
I have Facing This issue
$ next build
Warning: Built-in CSS support is being disabled due to custom CSS configuration being detected.
See here for more info: https://err.sh/next.js/built-in-css-disabled

Creating an optimized production build  

Failed to compile.

./node_modules/@fortawesome/fontawesome-free/css/all.min.css
ModuleParseError: Module parse failed: Unexpected character '' (1:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
(Source code omitted for this binary file)


> Build error occurred
Error: > Build failed because of webpack errors
    at build (/var/ezdev/employzone-us/node_modules/next/dist/build/index.js:13:900)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Can you please help me.

**Comment by trungbui2000** [comment]
Hey I am running into the same issue as well and was wondering if anyone had found a solution/work-around for this issue yet?
